modules/configuring-network-overrides-ibmz.adoc

```diff
@@ -0,0 +1,42 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="configuring-network-overrides-ibm"]
+= Configuring network overrides in {ibm-z-title}
+include::_attributes/common-attributes.adoc[]
+:context: configuring-network-overrides-ibm
+
+You can specify a static IP address on {ibm-z-title} machines that uses Logical Partition (LPAR) and z/VM. This is useful when the network devices do not have a static MAC address assigned to them. 
+
+.Procedure
+
+. If you have an existing `.parm` file, edit it to include the following entry:
++
+[source,terminal]
+----
+ai.ip_cfg_override=1
+----
+
+This parameter allows the file to add the network settings to the CoreOS installer.
+
+.Example `.parm` file
+[source,terminal]
+----
+rd.neednet=1 cio_ignore=all,!condev
+console=ttysclp0
+coreos.live.rootfs_url=<coreos_url> <1>
+ip=<ip>::<gateway>:<netmask>:<hostname>::none nameserver=<dns> 
+rd.znet=qeth,<network_adaptor_range>,layer2=1
+rd.<disk_type>=<adapter> <2> 
+rd.zfcp=<adapter>,<wwpn>,<lun> random.trust_cpu=on <3>
+zfcp.allow_lun_scan=0 
+ai.ip_cfg_override=1
+ignition.firstboot ignition.platform.id=metal 
+random.trust_cpu=on
+----
+<1> For the `coreos.live.rootfs_url` artifact, specify the matching `rootfs` artifact for the `kernel` and `initramfs` that you are booting. Only HTTP and HTTPS protocols are supported.
+<2> For installations on direct access storage devices (DASD) type disks, use `rd.` to specify the DASD where {rhel-first} is to be installed. For installations on Fibre Channel Protocol (FCP) disks, use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {rhel} is to be installed.
+<3> Specify values for `adapter`, `wwpn`, and `lun` as in the following example: `rd.zfcp=0.0.8002,0x500507630400d1e3,0x4000404600000000`.
```

Comment:
Just a note that this example still has the issues of 1. using user-replaced values even though it's called an example (this can be addressed later) and 2. having callout labels that don't seem relevant at all to the procedure or the step:
```suggestion
rd.neednet=1 cio_ignore=all,!condev
console=ttysclp0
coreos.live.rootfs_url=<coreos_url>
ip=<ip>::<gateway>:<netmask>:<hostname>::none nameserver=<dns> 
rd.znet=qeth,<network_adaptor_range>,layer2=1
rd.<disk_type>=<adapter>
rd.zfcp=<adapter>,<wwpn>,<lun> random.trust_cpu=on 
zfcp.allow_lun_scan=0 
ai.ip_cfg_override=1
ignition.firstboot ignition.platform.id=metal 
random.trust_cpu=on
----
```

Reply:
@skopacz1 I would look into it in the next PR. As this requires more discussion with the team. Pls allow me time until then. Thank you!